dashboard/config/scripts/levels/poetry_hoc_exemplar2.level

```diff
@@ -40,14 +40,17 @@
       "poem art predict_library"
     ],
     "preload_asset_list": null,
+    "encrypted_examples": [
+
+    ],
```

Comment:
@code-dot-org/curriculum-platform it looks innocuous, but is an empty array for `encrypted_examples` expected here? i would've assumed that the value would either be `null` or an encrypted string (like the change to `poetry_hoc_spriteSizes.level` below), but i'm not super familiar with this so i want to double-check

Reply:
good question Maddie! it's a bit strange that this diff is showing up, especially given that this level has been edited recently. overall this looks like a cosmetic change to me and I wouldn't be worried about it breaking anything, though.